src/lib.rs

```diff
@@ -46,4 +46,29 @@ mod trie;
 
 pub use db::{MemoryDB, DB};
 pub use errors::{MemDBError, TrieError};
+pub use hasher::Hasher;
 pub use trie::{PatriciaTrie, Trie};
+
+use std::sync::Arc;
+use trie::TrieResult;
+
+pub fn verify_proof<H: Hasher>(
```

Comment:
standalone function should not stand alone here, should set in you project's util.

Reply:
Noop, this is a very useful interface. For most users, they don't know how to develop a standalone validation function without reading the source code. So that they will think it is impossible to validate a proof without a complete MPT. According to the design in [parity-trie](https://docs.rs/trie-db/0.24.0/trie_db/proof/fn.verify_proof.html), we should provide a standalone verify function.

Reply:
So we also need give standalong **generate_proof**？ But cita-trie is cita's trie why we should follow paritfy-trie closely which need too much work to few maintainers. And a little temp work can't make sense? Why we need do it?

Reply:
For cita_tire, validating proofs is a common interface. However, the current interface arguments give users the false perception that they can only verify if they have the entire MPT. In fact, verification is possible with root, proof, and hash function. Developers wanting to discover this need to read the source code, which is very unfriendly.

Reply:
If you persist in, at least not write in lib.rs add a new mod plz.